src/components/Timestamp.tsx

```diff
@@ -48,9 +48,9 @@ export class Timestamp extends React.Component<Props, State> {
   }
 
   render() {
-    const t = moment(this.props.timestamp)
+    const t = moment.utc(this.props.timestamp)
```

Comment:
The only reason i'm asking about the UTC stuff is because that must have always been wrong (if only by a few hours)